irc/client.go

```diff
@@ -176,12 +176,12 @@ func NewClient(server *Server, conn net.Conn, isTLS bool) {
 		client.Notice(client.t("*** Looking up your username"))
 		resp, err := ident.Query(clientHost, serverPort, clientPort, IdentTimeoutSeconds)
 		if err == nil {
-			username := resp.Identifier
-			cfusername, err := CasefoldName(username)
-			if err == nil {
+			ident := resp.Identifier[:config.Limits.IdentLen-1]
```

Comment:
I believe this works in Python but not in golang: `panic: runtime error: slice bounds out of range`

Reply:
fix'd, thanks for the yell here